clients/csharp-dotnet-core/voice-assistant-test/tool/BotConnector.cs

```diff
@@ -148,6 +153,11 @@ public void InitConnector(AppSettings settings)
                 }
             }
 
+            if (this.appsettings.RealTimeAudio)
+            {
+                config.SetProperty("SPEECH-AudioThrottleAsPercentageOfRealTime", "100");

```

Comment:
There was a 2nd property mentioned in the e-mail attachement I forwarded you. Shouldn't both be set?

Reply:
Done